src/components/Log_Menu.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+interface Props {
+  logFilterValue: number; 
+  onLogFilterChange: (newLogFilterValue: number) => void; 
+}
+
+const BasicSelect: React.FC<Props> = ({ logFilterValue, onLogFilterChange }) => {
+  const handleChange = (event: SelectChangeEvent) => {
+    const newLogFilterValue = event.target.value as unknown as number;
+    onLogFilterChange(newLogFilterValue); 
+  };
+
+  return (
+    <Box sx={{ minWidth: 120 }}>
+      <FormControl fullWidth>
+        <InputLabel id="log-filter-id">Minimum Log Filter</InputLabel>
+        <Select
+          labelId="log-filter-id"
+          id="log-filter-label"
+          value={logFilterValue.toString()}
+          label="Minimum Log Filter"
+          onChange={handleChange}
+        >
+          <MenuItem value={7}>DEBUG</MenuItem>
+          <MenuItem value={6}>INFO</MenuItem>
+          <MenuItem value={5}>NOTICE</MenuItem>
+          <MenuItem value={4}>WARNING</MenuItem>
+          <MenuItem value={3}>ERROR</MenuItem>
+          <MenuItem value={2}>CRITICAL</MenuItem>
+          <MenuItem value={1}>ALERT</MenuItem>
+          <MenuItem value={0}>EMERGENCY</MenuItem>
```

Comment:
There's duplication here with the `logLevel` function. I suggest you store this information is a common data structure somewhere and programatically populate both this and the `logLevel` function based on that.